src/dev/flang/fuir/FUIR.java

```diff
@@ -1041,7 +1041,12 @@ else if (_doesNotNeedCode.get(cl - CLAZZ_BASE))
         var result = switch (clazzKind(cc))
           {
           case Abstract, Choice -> false;
-          case Intrinsic, Routine, Field -> (cc.isInstantiated() || cc.feature().isOuterRef()) && cc != Clazzes.conststring.getIfCreated() && !cc.isAbsurd();
+          case Intrinsic, Routine, Field ->
+            (cc.isInstantiated() || cc.feature().isOuterRef())
+            && cc != Clazzes.conststring.getIfCreated()
+            && !cc.isAbsurd()
+            // NYI: this should not depend on string comparison!
+            && !(cc.feature().qualifiedName().equals("void.absurd"));
```

Comment:
@fridis We are in clazzNeedsCode() here. I want to return false for any arguments of void features. How can I express this without the hack above?